web/packages/teleterm/src/ui/services/deepLinks/deepLinksService.ts

```diff
@@ -88,13 +88,70 @@ export class DeepLinksService {
         break;
       }
       case '/authenticate_web_device': {
-        // TODO (avatus): add authenticate device web confirmation dialog
-        // this case is a stub and will not be reached
+        const id = result.url?.searchParams?.id;
+        const token = result.url?.searchParams?.token;
```

Comment:
We guarantee that `id` and `token` is set in `deepLinks.ts`, no? (I mean that we should remove the optional chaining)

Reply:
I tend to add optional chaining just because I get scared when doing deep nested fields but you're right, it should be safe at this point in the flow